azerothcore/azerothcore-wotlk
# fix(Core/BattlegroundAV): spawn generic gameobjects in Alterac Valley

I've been "collecting" these in the DB for a while, now it's time to move these spawns to the AV script and actually spawn them.



## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



.debug bg 1
join Alterac Valley
wait for battleground start (2min)
observe newly spawned gameobjects
```
// Burning Embers
.go xyz -1235.57  -241.478  73.4377    
.go xyz -1248.85   -254.06  73.4377    
.go xyz -1248.15  -245.599  73.4377    
.go xyz -1237.21  -260.168  73.4377    
.go xyz -1260.37  -248.767  77.9454    
.go xyz -1249.32  -244.907  92.3372    
.go xyz -1250.09  -254.604  92.3015    
// Wooden Chair
.go xyz 41.1672  -426.866  44.6828     
.go xyz 39.0988  -425.746   44.688     
.go xyz 25.2482  -433.104  47.6369     
.go xyz  25.758  -425.837  47.6369     
.go xyz 27.6786   -427.69  47.6369     
// Anvil
.go xyz  -100.071  -583.512  41.7316 
.go xyz  -102.733  -588.775  41.7316   
.go xyz  -100.966  -586.958  41.7316   
.go xyz  -106.172  -589.721  41.7316   
.go xyz   -1245.2  -307.059  63.3199   
.go xyz  -1258.23  -310.977  63.2015   
.go xyz    360.38  -527.549  71.1205   
.go xyz   646.207  -57.2431  41.6587   
.go xyz   548.682  -100.846  44.8191   
.go xyz  -172.363  -452.824  33.2796   
.go xyz   552.429  -102.764  44.8191   
.go xyz  -171.282  -456.892  33.2796   
.go xyz   549.472  -101.547  44.8191   
.go xyz  -172.356   -453.88  33.2796   
.go xyz   550.347  -101.698  44.8191   
.go xyz  -171.882  -454.632  33.2796   
.go xyz   551.271  -102.226  44.8191   
.go xyz  -171.656  -455.671  33.2796   
.go xyz  -170.699   -458.41  33.2796   
.go xyz   553.947  -103.347  44.8191   
.go xyz   -170.14  -457.609  33.2796   
.go xyz   553.725  -102.396  44.8191   
// Forge
.go xyz  -105.423  -584.352  42.0149   
.go xyz   361.097  -531.693  71.1898   
.go xyz   649.264  -59.1111  41.5476   
// Bonfire
.go xyz   360.915  -497.099  71.3389   
.go xyz  -743.427  -398.241  76.4266   
// Brazier
.go xyz  -1290.27   -172.47  72.1853   
.go xyz  -1099.21  -266.231  57.8849   
.go xyz   -1082.6  -266.681  57.8575   
.go xyz  -1140.82  -343.392  50.9077   
.go xyz  -1139.68  -356.288   51.264   
.go xyz  -1215.59  -371.946  56.5293   
.go xyz  -1241.25  -345.115  59.6867   
.go xyz  -1202.81  -271.599  72.5805   
.go xyz  -1282.21  -284.083   87.256   
.go xyz  -1256.55  -280.278  73.9473   
.go xyz  -1423.16  -318.436  89.1136   
// Campfire
.go xyz  -1286.25  -184.481  71.8334   
.go xyz  -1280.17  -220.536  72.2686   
.go xyz  -375.017   -523.71  84.0771   
.go xyz   -413.51  -521.413  84.0771   
.go xyz  -487.401  -526.486  82.5708   
.go xyz  -251.556  -562.061  68.1725   
.go xyz  -353.118   37.0955  65.8904   
.go xyz  -1070.53  -617.908  51.3249   
// Cauldron
.go xyz   12.8247   -120.21  41.6669   
// Chair
.go xyz  -145.341  -444.846  26.4163   
.go xyz   560.834  -75.4266  37.9558   
.go xyz   550.678  -79.8234  44.8257   
.go xyz  -155.405   -440.24  33.2862   
.go xyz   568.326  -89.4992  37.9558   
.go xyz  -150.787  -459.829  26.4163   
.go xyz   550.597  -77.5213  44.8257   
.go xyz  -153.748  -438.639  33.2862   
.go xyz   570.419  -89.0567  37.9558   
.go xyz  -149.057  -461.089  26.4163   
.go xyz   555.517  -101.589  44.8257   
.go xyz  -168.342    -458.4  33.2862   
.go xyz   561.851  -73.1481  37.9558   
.go xyz  -142.968  -444.076  26.4163   
// Wanted: ORCS!
.go xyz  618.748 -52.1126 42.1122
// Wanted: DWARVES!
.go xyz -1181.10 -370.747 53.6246
```

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
